agent/app/agent_capability_test.go

```diff
@@ -751,3 +765,151 @@ func TestCapabilitesScanPluginsErrorCase(t *testing.T) {
 		}
 	}
 }
+
+func TestCapabilitesExecuteCommandShouldBeAddedWhenDependenciesExist(t *testing.T) {
```

Comment:
Can we use a test table instead of two test functions? `TestCapabilitesExecuteCommandShouldBeAddedWhenDependenciesExist` and `TestCapabilitesExecuteCommandShouldNotBeAddedWhenDependenciesDoNotExist` have pretty much the same code.

Reply:
sure, change to test table